### PR TITLE
Adding An Event Video Page to Docs

### PR DESCRIPTION
**Pull request by kodebae**
## Adding An Event Video Page to Docs
* The purpose of this PR is to add a file where we can link back to WWCode Greenville online events.
* The suggested change solves the issue of not having a file for this before.
* I am requesting the following feedback from @klmansel 

### Type of Change
Added a Markdown file with links to YT content.

### How Has This Been Tested?
Manually tested each link.

### CODE REVIEW

**Comment by klmansel** [comment]
@kodebae can you resolve the merge conflicts? Maybe your `main` branch was behind?

File: README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@
 
 <!-- TABLE OF CONTENTS -->
 ![Table of Contents][table-of-contents-heading]
-  
 
 * [About This Project](#about-this-project)
 * [Hacktoberfest 2023](#hacktoberfest)
   * [Prerequisites](#prerequisites)
   * [Installation](#installation)
+  * [PR Template](#pr-template)
```
Comment (klmansel) [inline README.md:18]: is this from another PR?

File: README.md [hunk 2]
```diff
@@ -64,6 +64,33 @@ There are two ways to use git to interact with our repository: command-line Git
...
+### Screenshots (If Applicable)
+<p>Screenshot 1</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+
+<p>Screenshot 2</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+
+<p>Screenshot 3</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+-->
```
Comment (klmansel) [inline README.md:92]: This is the PR template - I see that now. Did you mix 2 issues maybe?

File: WWCode_Event_Recordings/EventRecordings.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Women Who Code Greenville YouTube Videos:
+*You can find a list of all of WWCodes videos [here](https://www.youtube.com/@WomenWhoCode/) on our YouTube channel. Don't forget to subscribe for future updates.*
```
Comment (klmansel) [inline WWCode_Event_Recordings/EventRecordings.md:2]: our network YouTube url is https://www.youtube.com/playlist?list=PLVcEZG2JPVhclL1RUkH6Dt0nE3CXFcXpj

Comment (klmansel) [reply WWCode_Event_Recordings/EventRecordings.md:2]: There are a few more videos there

**Review by klmansel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kodebae) [reply README.md:18]: Yes I had already added code for this when I created the PR.

Comment (kodebae) [reply README.md:18]: Not sure what happened I think I added code before I switched to a new branch. Sorry about that.

Comment (klmansel) [reply README.md:18]: @kodebae can you remove this code from this branch?

File: README.md (hunk 2 above)
Comment (klmansel) [reply README.md:94]: @kodebae can you remove this from this branch too?

**Review by klmansel — CHANGES_REQUESTED** [review]
almost! can you remove the code that doesn't belong to this PR?

File: WWCode_Event_Recordings/EventRecordings.md (hunk 3 above)
Comment (klmansel) [inline WWCode_Event_Recordings/EventRecordings.md:2]:
```suggestion
*You can find a list of all of WWCode videos [here](https://www.youtube.com/playlist?list=PLVcEZG2JPVhclL1RUkH6Dt0nE3CXFcXpj) on our YouTube channel. Don't forget to subscribe for future updates.*
```

Comment (kodebae) [reply WWCode_Event_Recordings/EventRecordings.md:2]: I checked the link. I belive I linked to these videos already. But thank you for letting me know.

File: README.md (hunk 2 above)
Comment (kodebae) [reply README.md:94]: Yes, I am taking this out now. I was not aware that we needed to add this directly into GH. I thought that I was supposed to add the markdown directly to the README.md. I apologize for the confusion on my part.

File: .DS_Store
Comment (klmansel) [inline .DS_Store:1]: Please remove this file as well. You can actually add it to `.gitignore` so this won't be an issue in every PR (not just yours)

**Review by klmansel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kodebae) [reply README.md:18]: done. ;)

Comment (klmansel) [reply README.md:18]: The Readme still has the PR template stuff in it 🤔

File: .DS_Store
Comment (kodebae) [reply .DS_Store:1]: It seems like this has been resolved on my end somehow. I am seeing .DS_store in my .gitignore right now.

Comment (klmansel) [reply .DS_Store:1]: Cool! Your other PR has been merged - once you open a new PR we can get this one in!

Comment (klmansel) [reply .DS_Store:1]: (Or reopen this one)

**Pull request closed without merging**